MekHQ/src/mekhq/campaign/parts/FireControlSystem.java

```diff
@@ -72,15 +72,53 @@ public void updateConditionFromEntity(boolean checkForDestruction) {
 			hits = ((Aero)unit.getEntity()).getFCSHits();
 			if(checkForDestruction 
 					&& hits > priorHits 
-					&& Compute.d6(2) < campaign.getCampaignOptions().getDestroyPartTarget()) {
+					&& (hits < 3 && !campaign.getCampaignOptions().useAeroSystemHits())
+                    && Compute.d6(2) < campaign.getCampaignOptions().getDestroyPartTarget()) {
 				remove(false);
 				return;
-			}
+			} else if (hits >= 3) {
+                remove(false);
+                return;
+            }
 		}
 	}
 	
 	@Override 
 	public int getBaseTime() {
+        if (campaign.getCampaignOptions().useAeroSystemHits()) {
+            //Test of proposed errata for repair times
+            Entity e = unit.getEntity();
+            if (isSalvaging()) {
+                if (e.hasETypeFlag(Entity.ETYPE_DROPSHIP) || e.hasETypeFlag(Entity.ETYPE_JUMPSHIP)) {
+                    if (e.hasNavalC3()) {
+                        return 8640;
```

Comment:
Instead of a fixed number, we should have the base and then have a multiplier for Naval C3.

Reply:
In fact, it looks like Naval C3 is always double? If so, I would have this code look like this (Everywhere Naval C3 has an impact):
    int time = 240;
    if (e.hasNavalC3()) {
        time *= 2;
    }
    return time;